freud/box.pyx

```diff
@@ -341,6 +341,27 @@ cdef class Box:
 
         return np.squeeze(vecs) if flatten else vecs
 
+    def center(self, vecs):
+        R"""Center an array of vectors using periodic boundaries.
+
+        Args:
+            vecs (:math:`\left(3, \right)` or :math:`\left(N, 3\right)` :class:`numpy.ndarray`):
+                Vector(s) to be unwrapped.
+
+        Returns:
+            :math:`\left(3, \right)` or :math:`\left(N, 3\right)` :class:`numpy.ndarray`:
+                Centered vector(s).
+        """  # noqa: E501
+        vecs = np.asarray(vecs)
+        vecs = np.atleast_2d(vecs)
```

Comment:
Be careful here with 1D vectors. If a single vector is passed in, I don't think this is returning a single vector. It's also not very useful to center one point, so I'd be fine with requiring a 2D array of input data. If you make that change, please alter the docstrings to only allow (N, 3) for inputs and outputs.

Reply:
Thanks for catching it. I removed the 2D enforcement and let `make_absolute` handle it when returning.